esp-hal-common/src/embassy/executor/interrupt.rs

```diff
@@ -61,6 +69,7 @@ macro_rules! from_cpu {
     };
 }
 
+// from_cpu!(0); // reserve 0 for thread mode & multi-core
```

Comment:
We can cfg this with `single_core` right? Or do we need the the interrupt even on single cores?

Reply:
No but it would be pretty messy IMO, esp. in __pender

Reply:
I'm going to go ahead and merge this, but if one of you could open an issue for this please that'd be great!

Reply:
Oh, too slow 😁

Reply:
https://github.com/esp-rs/esp-hal/issues/969